editor/editor_inspector.cpp

```diff
@@ -427,6 +427,7 @@ void EditorProperty::_set_read_only(bool p_read_only) {
 void EditorProperty::set_read_only(bool p_read_only) {
 	read_only = p_read_only;
 	_set_read_only(p_read_only);
+	GDVIRTUAL_CALL(_set_read_only, p_read_only);
```

Comment:
Shouldn't it supersede the virtual `_set_read_only()` if the `GDVIRTUAL` one is defined? That's how it works usually.
```cpp
if (GDVIRTUAL_CALL(_set_read_only, p_read_only) {
	return;
}
_set_read_only(p_read_only);
```